app/jobs/regular/change_display_name.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Jobs
+  class ChangeDisplayName < ::Jobs::Base
+    sidekiq_options queue: "low"
+
+    def execute(args)
+      @user = User.find_by(id: args[:user_id])
+
+      return unless user.present?
+
+      # We need to account for the case where the instance allows
+      # name to be empty by falling back to username.
+      @old_display_name = (args[:old_name].presence || user.username).unicode_normalize
+      @new_display_name = (args[:new_name].presence || user.username).unicode_normalize
+
+      @quote_rewriter = QuoteRewriter.new(user.id)
+
+      update_posts
+      update_revisions
+    end
+
+    private
+
+    attr_reader :user, :old_display_name, :new_display_name, :quote_rewriter
+
+    def update_posts
+      Post
+        .with_deleted
+        .joins(quoted("posts.id"))
+        .where("p.user_id = :user_id", user_id: user.id)
+        .find_each { |post| update_post(post) }
+    end
+
+    def update_revisions
+      PostRevision
+        .joins(quoted("post_revisions.post_id"))
+        .where("p.user_id = :user_id", user_id: user.id)
+        .find_each { |revision| update_revision(revision) }
+
+      PostRevision
+        .joins(quoted("post_revisions.post_id"))
+        .where("p.user_id = :user_id", user_id: user.id)
+        .find_each { |revision| update_revision(revision) }
```

Comment:
```suggestion
      PostRevision
        .joins(quoted("post_revisions.post_id"))
        .where("p.user_id = :user_id", user_id: user.id)
        .find_each { |revision| update_revision(revision) }
```

I think we accidentally duplicated the same block of code here?

Reply:
Good catch! It was a bad git conflict resolution. 👌